index.html

```diff
@@ -1,222 +0,0 @@
-            <a class="dropdown-item smooth text-uppercase hidden-sm-up" href="#atlas">Atlas</a>
-            <a class="dropdown-item smooth text-uppercase hidden-md-up" href="#thingsToDo">Things to do </a>
-            <a class="dropdown-item smooth text-uppercase hidden-md-up" href="https://my-first-store-that-is-so-awesome.myshopify.com/">Gear</a>
-            <a class="dropdown-item text-uppercase smooth" href="#globalService">Global Service</a>
-            <a class="dropdown-item text-uppercase smooth" href="#">SM/AC map</a>
-            <a class="dropdown-item text-uppercase" href="https://aswwu.com/uploads/ASWWU/ASWWU-Application-2016-2017.docx">Work for ASWWU</a>
-          </div>
-        </li>
-      </ul>
-    </nav>
-
-    <!-- Offset navbar. -->
-    <div id="top" style="width:100%;height:74px;"></div>
-
-    <!-- Video -->
-    <!-- <div class="embed-responsive embed-responsive-16by9 full_height_parent">
-      <iframe class="embed-responsive-item full_height_child" src="https://player.vimeo.com/video/162930740?autoplay=1&loop=1&color=ffffff&title=0&byline=0&portrait=0" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-    </div> -->
-    <!-- ASWWU Logo -->
-    <div id="aswwuLogo">
-      <img src="https://aswwu.com/media/images/ASWWU_logo_full.png" />
-    </div>
-
-    <!-- About US -->
-    <section id="aswwu">
-      <div class="container">
-        <h1 class="text-xs-center text-uppercase">ASWWU</h1>
-        <hr>
-        <h5 class="text-xs-center text-uppercase">We are aswwu, and so are you.</h5><br />
-        <div class="row ">
-          <div class="col-md-6 offset-md-3">
-            <p>Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. Separated they live in Bookmarksgrove right at the coast of the Semantics, a large language ocean.</p>
-
-            <p>A small river named Duden flows by their place and supplies it with the necessary regelialia. It is a paradisematic country, in which roasted parts of sentences fly into your mouth.</p>
-          </div>
-    </section>
-
-    <!-- Mask -->
-    <div id="mask" class="parallax-window center-content" style="background-image: url('https://static.wixstatic.com/media/de317edf9480039c6c4d425f78b91e1c.png/v1/crop/x_0,y_0,w_330,h_331/de317edf9480039c6c4d425f78b91e1c.png'); background-repeat: repeat;background-size:initial; background-color:#99d7ea;">
-      <div class="container">
-        <h1 class="text-white text-uppercase text-giant">The mask </h1><br />
-        <div id="searchInputGroup" class="input-group">
-          <input id="maskSearchQuery" type="text" class="form-control" placeholder="Find the one...">
-          <span class="input-group-btn">
-            <button class="btn btn-secondary btn-outline-secondary giant-link giant-white" type="button" onClick="maskSearch()">Go!</button>
```

Comment:
`maskSearch()` isn't defined at the moment. I'm sure it will get defined at some point.